CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ to view all changes since the `0.2.1` release.
 
 ### Added
 
+- [`Mesh` overhaul with custom vertex attributes][599]
+    - Any vertex attribute can now be added over `mesh.attributes.insert()`.
+    - See `example/shader/mesh_custom_attribute.rs`
 - [Touch Input][696]
 - [Do not depend on spirv on wasm32 target][689]
 - [Another fast compile flag for macOS][552]
```

Comment:
Oh, would you mind removing these two? These were also accidentally duplicated by the mesh pr, but I missed them in https://github.com/bevyengine/bevy/pull/755